sqiffy-symbol-processor/src/main/kotlin/com/dzikoysk/sqiffy/processor/generators/EntityGenerator.kt

```diff
@@ -118,4 +126,24 @@ class EntityGenerator(private val context: KspContext) {
                     .build()
             )
 
+    private fun String.toKotlinCode(dataType: DataType, typeName: TypeName): String {
+        return when (dataType) {
+            /* Special types */
+            DataType.UUID_TYPE -> "UUID.fromString(\"$this\")"
+            DataType.ENUM -> "$typeName.$this"
+
+            /* Regular types */
+            DataType.CHAR -> "'$this'"
+            DataType.VARCHAR, DataType.TEXT -> "\"$this\""
+            DataType.BINARY -> "\"$this\".toByteArray()"
+            DataType.BOOLEAN -> this.toBoolean()
+            DataType.LONG -> "${this}L"
+            DataType.FLOAT -> "${this}F"
+            DataType.DATE -> "LocalDate.parse(\"$this\")"
+            DataType.DATETIME -> "LocalDateTime.parse(\"$this\")"
+            DataType.TIMESTAMP -> "Instant.parse(\"$this\")"
+            else -> this
+        }.toString()
```

Comment:
You should make sure each case returns string instead of calling `.toString()` on whole when statement.